src/aggregate/expr/lexer.c

```diff
@@ -1,11 +1,11 @@
 
 /* #line 1 "lexer.rl" */
-/*
- * Copyright Redis Ltd. 2016 - present
- * Licensed under your choice of the Redis Source Available License 2.0 (RSALv2) or
- * the Server Side Public License v1 (SSPLv1).
- */
-
+/*
+ * Copyright Redis Ltd. 2016 - present
+ * Licensed under your choice of the Redis Source Available License 2.0 (RSALv2) or
+ * the Server Side Public License v1 (SSPLv1).
+ */
+
 #include <stdlib.h>
```

Comment:
It seems like `strtod` comes from `#include <stdlib.h>`
Maybe we can remove it in some places?
If you get a bunch of compilation errors then don't do it.